boa_engine/src/builtins/date/mod.rs

```diff
@@ -1649,6 +1817,25 @@ impl Date {
         }
     }
 
+    /// `Date.prototype.toLocaleDateString()`
+    ///
+    /// The `toLocaleDateString()` method returns the date portion of the given Date instance
+    /// according to language-specific conventions.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-date.prototype.tolocaledatestring
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
```

Comment:
This link seems wrong, same with all the other toLocale* mdn links

Reply:
Sorry, but what exactly is wrong with them? They work, link the correct thing and uses the same format as other functions.

Also, as `locale*` are out of this PR's scope, should I remove the docs or keep them?

Reply:
True, I missread that. I answered the question that below.